### PR TITLE
Code cleanup based on ESLint warnings

### PR DESCRIPTION
**Pull request by Turbo87**
The contents in this PR were mostly extracted from #13549

/cc @rwjblue @stefanpenner

### CODE REVIEW

File: packages/ember-glimmer/tests/integration/components/curly-components-test.js [hunk 1]
```diff
@@ -2224,16 +2218,10 @@ moduleFor('Components test: curly components', class extends RenderingTest {
   }
 
   ['@test specifying classNames results in correct class'](assert) {
-    let clickyThing;
-
     this.registerComponent('some-clicky-thing', {
       ComponentClass: Component.extend({
         tagName: 'button',
         classNames: ['foo', 'bar'],
```
Comment (rwjblue) [inline packages/ember-glimmer/tests/integration/components/curly-components-test.js]: Need to remove this trailing comma

File: packages/ember-glimmer/tests/integration/components/curly-components-test.js [hunk 2]
```diff
@@ -2261,15 +2249,10 @@ moduleFor('Components test: curly components', class extends RenderingTest {
   }
 
   ['@test specifying custom concatenatedProperties avoids clobbering'](assert) {
-    let clickyThing;
     this.registerComponent('some-clicky-thing', {
       ComponentClass: Component.extend({
         concatenatedProperties: ['blahzz'],
         blahzz: ['blark', 'pory'],
```
Comment (rwjblue) [inline packages/ember-glimmer/tests/integration/components/curly-components-test.js]: trailing comma

**Review by rwjblue — CHANGES_REQUESTED** [review]
Looks good, happy to land independently.

Can you fixup the two issues that JSCS caught, and squash the commits?

**Comment by Turbo87** [comment]
@rwjblue done

**Comment by Turbo87** [comment]
also fixed the failing JSHint test now... Travis will hopefully agree :)

**Comment by Turbo87** [comment]
🍏

**Review by rwjblue — APPROVED** [review]

**Comment by rwjblue** [comment]
Still need to squash commmits though

**Comment by Turbo87** [comment]
@rwjblue not a fan of squashing in general, but: done!

**Comment by rwjblue** [comment]
Thanks!

**Pull request merged**